CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## unreleased
 
+* Updated `@enact/dev-utils` version to `^6.0.0`.
```

Comment:
Please modify your changelog to be brief like the below link.
https://github.com/enactjs/cli/pull/321/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed